root/static/scripts/guess-case/utils.js

```diff
@@ -144,8 +152,8 @@ exports.titleString = function (is, forceCaps) {
     }
 
     var os;
-    var lc = is.toLowerCase();
```

Comment:
Yes. For illustration, I did it in this branch: https://github.com/metabrainz/musicbrainz-server/compare/master...uklauer:pr-410